pkg/cmd/updatecharts/charts.go

```diff
@@ -153,64 +153,56 @@ func updateLocalChart(ctx context.Context, name, url string, entries []Entry, v
 			continue
 		}
 		if v.Equal(ev) {
-			// download and extract
 			if len(entry.Urls) == 0 {
 				return fmt.Errorf("no urls found for %s", entry.Version)
 			}
 			chartSourceUrl := entry.Urls[0]
 
+			outputPath := fmt.Sprintf("./charts/%s/%s", name, ev.String())
+			if err := os.MkdirAll(outputPath, 0755); err != nil {
+				return fmt.Errorf("failed to create output directory: %w", err)
+			}
+
+			// Handle OCI registry URLs
+			if strings.HasPrefix(chartSourceUrl, "oci://") {
+				return handleOCIChart(ctx, chartSourceUrl, name, ev.String(), outputPath)
```

Comment:
pending support for OCI registries in Wayfinder, we could just *skip* any OCI ones for now, to get this merged and get the workflow working once more?